### DISCUSSION TITLE
Help understanding the get_world_collision_distance_from_joints api

### DISCUSSION THREAD

**Discussion by shannonwarren** [NVlabs/curobo#254] [Software Q&A] [answered]
I use the sample code to print self collisions and world collisions

```
q_s = curobo_fn.sample(6, mask_valid=False)
d_world, d_self = curobo_fn.get_world_self_collision_distance_from_joints(q_s)
```

what's q_s here and the number 6? I chose the number 6 since my arm has 6 joints but I don't think that's right.

**Comment by balakumar-s** [comment, marked as answer]
q_s is a batch of joint configurations. To check collisions for 10 joint configurations, you would use size of [10,dof]. The sample function is just a helper function to sample `n` different joint configurations for your robot. Here we sample 6 joints configurations.
